### PR TITLE
Clean up user guide for submission

### PR DESCRIPTION
**Pull request by uniqly**
Reformatted some sections to it more pleasant to use

### CODE REVIEW

**Review by blanklogic — APPROVED** [review]
LGTM! Great clean up of the User Guide

File: docs/UserGuide.md [hunk 1]
```diff
@@ -206,50 +237,59 @@ not** put any parameters or prefixes that are not recognised by the command.
...
 
+* Adds a company with the corresponding `NAME` and `EMAIL` to the list of companies.
+
 <box type="tip" seamless>
 
 **Tip 1:** A company can have any number of tags (including 0) <br>
 **Tip 2:** Tags can be a useful way to categorise different companies into various groups! <br>
-**Tip 3:** A company requires a name and email at the minimum
+**Tip 3:** A company requires a name and email at the minimum. Other parameters like `PHONE_NUMBER`, `ADDRESS` and `TAG` 
+can be added to company later on using the `edit` command.
```
Comment (realqijun) [inline docs/UserGuide.md:292]: Great tip!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -273,9 +309,14 @@ Examples:
 ![add a company](images/addCommand.png)
 > Result after executing `add n/Netflix e/contact@netflix.com p/4085403700 a/100 Winchester Circle, Los Gatos, CA`
 
+[back to command summary](#command-summary)
+[back to features](#features)
+[back to company commands](#company-commands)
```
Comment (realqijun) [inline docs/UserGuide.md:314]: Links to multiple sections are nice and consistent!

**Review by realqijun — APPROVED** [review]
LGTMMMMM

**Pull request merged**